### PR TITLE
Actually stream prevtx: required for v8.0.0

### LINKED ISSUE #26: Fix up BTC signing for v8.0.0, streaming prevtx
Since 8.0.0 the previous tx of each input must be streamed. The JS package and demo need to be fixed up for this.

### PR DESCRIPTION
**Pull request by benma**
Closes #26

### CODE REVIEW

**Comment by x1ddos** [comment]
Updated the title and description. Did I get it right?

File: demo/demo.js [hunk 1]
```diff
@@ -142,18 +142,52 @@ document.getElementById("btcAddressSimple").addEventListener("click", async () =
 function makeExampleTx(keypathAccount) {
     const inputs = [
         {
-            "prevOutHash": new Uint8Array(32).fill(49), // arbitrary constant
+            "prevOutHash": Uint8Array.from([0xc5, 0x8b, 0x7e, 0x3f, 0x12, 0x00, 0xe0, 0xc0, 0xec, 0x9a, 0x5e, 0x81, 0xe9, 0x25, 0xba, 0xfa, 0xce, 0x2c, 0xc1, 0xd4, 0x71, 0x55, 0x14, 0xf2, 0xd8, 0x20, 0x5b, 0xe2, 0x50, 0x8b, 0x48, 0xee]),
```
Comment (x1ddos) [inline demo/demo.js:145]: Is there an easy command go reconstruct this hash, in case I need to update the demo in the future?

Comment (x1ddos) [reply demo/demo.js:145]: Would be nice to have a comment.

File: gowrapper/main.go [hunk 2]
```diff
@@ -253,7 +305,7 @@ type btcSignOutputRequest struct {
 func (output *btcSignOutputRequest) toOutput() (*messages.BTCSignOutputRequest, error) {
 	int, ok := new(big.Int).SetString(output.Value, 10)
 	if !ok {
-		return nil, errors.New("expected decimal string as value")
+		return nil, errors.New("expected decimal string as value 1")
```
Comment (x1ddos) [inline gowrapper/main.go:308]: Looks like a leftover.

File: demo/demo.js (hunk 1 above)
Comment (benma) [reply demo/demo.js:145]: There is on "easy" one that I know of. When I added the demo to the firmware repo, I just printed what the firmware computed for it to construct a demo that works.

File: gowrapper/main.go (hunk 2 above)
Comment (benma) [reply gowrapper/main.go]: :+1:

**Pull request merged**